build.gradle.kts

```diff
@@ -20,14 +20,19 @@ tasks.compileTestJava {
 dependencies {
     implementation(platform("com.google.cloud:libraries-bom:24.3.0"))
     implementation("com.google.http-client:google-http-client-apache-v2")
-    implementation("com.google.api-client:google-api-client-gson:1.31.5")
+    implementation("com.google.http-client:google-http-client-gson")
```

Comment:
(more noob questions)
should we not pin versions?

Reply:
versions are in the `bom`